static/app/views/monitors/monitorHeaderActions.tsx

```diff
@@ -23,6 +24,10 @@ type Props = {
   orgId: string;
 };
 
+const DISABLED_TOOLTIP_TEXT = t(
+  'You must be an organization admin to perform this action'
```

Comment:
can't users with owner and manager roles edit too?

we have a similar check in in Dynamic Sampling https://github.com/getsentry/sentry/blob/68d6d2b517005456c131c6d1f18ce2ab632bbd8f/static/app/views/settings/project/dynamicSampling/dynamicSampling.tsx#L133

Reply:
ahh yup you're right will change the text here real quick